pkg/health/types.go

```diff
@@ -16,6 +16,36 @@
 
 package health
 
+import (
+	"fmt"
+	"github.com/InVisionApp/go-health"
+)
+
+// Settings type to be loaded from the environment
+type Settings struct {
+	FailuresTreshold int64 `mapstructure:"failures_treshold" description:"maximum failures in a row until component is considered down"`
+	Interval         int64 `description:"seconds between health checks of components"`
```

Comment:
change this to `time.Duration` and add `mapstructure`. if `time.Duration` adjust the description